Lib/inspect.py

```diff
@@ -2971,6 +2973,18 @@ def __init__(self, parameters=None, *, return_annotation=_empty,
                             # There is a default for this parameter.
                             seen_default = True
 
+                    if kind == _VAR_POSITIONAL:
+                        var_positional_count += 1
+                        if var_positional_count > 1:
+                            msg = 'more than one var positional parameter'
```

Comment:
Per [glossary](https://docs.python.org/3/glossary.html#term-parameter), probably it is better to use "var-positional" (and "var-keyword") terms.